aspnetcore/blazor/security/webassembly/additional-scenarios.md

```diff
@@ -1008,3 +1008,67 @@ Alternatively, the setting can be made in the app settings (`appsettings.json`)
 If tacking on a segment to the authority isn't appropriate for the app's OIDC provider, such as with non-AAD providers, set the <xref:Microsoft.AspNetCore.Builder.OpenIdConnectOptions.Authority> property directly. Either set the property in <xref:Microsoft.AspNetCore.Builder.JwtBearerOptions> or in the app settings file (`appsettings.json`) with the `Authority` key.
 
 The list of claims in the ID token changes for v2.0 endpoints. For more information, see [Why update to Microsoft identity platform (v2.0)?](/azure/active-directory/azuread-dev/azure-ad-endpoint-comparison).
+
+## Configure and use gRPC in components
+
+To configure a Blazor WebAssembly app to use the [ASP.NET Core gRPC framework](xref:grpc/index), register gRPC services for the app's message handler. The following example configures the app's authorization message handler to use the [`GreeterClient` service from the gRPC tutorial](xref:tutorials/grpc/grpc-start#create-a-grpc-service) (`Program.Main`):
```

Comment:
You need to mention that gRPC-Web needs to be enabled on the server.

This page talks about setting up gRPC-Web on the server - https://docs.microsoft.com/en-us/aspnet/core/grpc/browser?view=aspnetcore-3.1 (and Blazor client)